### DISCUSSION TITLE
Support nested fields with grouping statement

### DISCUSSION THREAD

**Discussion by mojerro** [directus/directus#15185] [Feature Requests]
ref: #15136 

 I try to use the aggregated function with the groupBy filter in the join table, and I found that the groupBy filter doesn't support nested fields which can generate group by SQL sentences with `JOIN` statement.

### What I want

Here is the Graphql code I expected.
```graphql
query MyQuery {
  consultations_aggregated(groupBy: {period_id: {duration: {_eq: ""}}}) {
    group
    count {
      id
    }
  }
  periods {
    id
    duration
  }
}
```

**Comment by noahjahn** [comment]
I ran into this same issue and spent too much time finding myself here.

@rijkvanzanten, it might be worth adding a warning about not being able to use nested fields in the documentation like on the Alias section: https://docs.directus.io/reference/query.html#aliases

**Comment by mojerro** [comment]
@rijkvanzanten hi, is there any plan to solve this? Or why do you think it's not important?

**Reply by rijkvanzanten** [reply]
We prioritize new features on community interest, and so far this improvement hasn't gotten a lot of traction

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
